scripts/power_control.py

```diff
@@ -114,6 +117,15 @@ def power_control(VCENTER_HOST,
         type=str,
         help='Password for given username on vcenter'
     )
+    parser.add_argument(
+        '--pattern ',
+        required=False,
+        default='sonar',
```

Comment:
Since this is a regex, then to have the same effect as `startswith('sonar-')` the default could be `r'^sonar-'`.